### ISSUE TITLE
Bar Graph Factorization Visualizer

### ISSUE THREAD

**Issue by katestange** [numberscope/frontscope#115]
A visualizer that shows factorizations of the sequence in an accessible, visual way.  Sequence is shown as a bar graph, one bar per term, where bars indicate factorization by stacking sub-bars of length log p for each prime p in the factorization (so the total bar length is log n where n is the term).  Colour coded in useful ways.  Pan/zoom.  Extra info appears upon mouseover/click (exact factorization, or highlight appearance of a particular prime number, etc.).

**Comment by gwhitney** [comment]
@katestange I think this is unblocked now. Not that there is any urgency, and right now might not be the best juncture, but whenever you like you could dust off the code you have for this and see if a PR could be generated.

**Comment by katestange** [comment]
Yeah right now probably isn't the best moment, but I did dust off the code last week and took a look at it.  So I'll be excited to do this when the time is right.

**Comment by gwhitney** [comment]
And this has just been merged into ui2 (#406). Yay! Closing.
